cluster-up/down.sh

```diff
@@ -9,4 +9,4 @@ fi
 
 source ${KUBEVIRTCI_PATH}hack/common.sh
 source ${KUBEVIRTCI_CLUSTER_PATH}/$KUBEVIRT_PROVIDER/provider.sh
-down
+down || true
```

Comment:
Would you be inclined to move the fix closer to root cause? (`cluster-up/cluster/kind/common.sh`

Reply:
I do think that `make cluster-down` should never cause entire test lanes to fail, but at the same time you're right, let's "fix" only what's broken for now! Pushed.